lib/middleware/query.js

```diff
@@ -15,6 +15,7 @@
 var merge = require('utils-merge')
 var parseUrl = require('parseurl');
 var qs = require('qs');
+var deprecate = require('depd')('query');
```

Comment:
This should be the name of the npm module; express in this instance

Reply:
@dougwilson - thanks for pointing this out, changed.